docs/root/version_history/current.rst

```diff
@@ -48,6 +48,7 @@ Bug Fixes
 * http: made the HeaderValues::prefix() method const.
 * jwt_authn: supports jwt payload without "iss" field.
 * rocketmq_proxy network-level filter: fixed an issue involving incorrect header lengths. In debug mode it causes crash and in release mode it causes underflow.
+* udp_proxy: fixed an issue with race condition. When the udp_proxy is activated and if hot restarting is triggered, sometimes it causes crash.
```

Comment:
Sorry what issue was this again? Usually we don't do release notes for small fixes like this but if we do do one it would be good to be a bit more specific.

Reply:
It was a fix that prevent envoy crash when hot reload is triggered.
It was caused because the udp listener was deleted first then read filter.
So, If there are some incoming packet from upstream, the Envoy try to pass packets to downstream via udp listener but there is no actual udp listener instance because it already deleted.
I hope you remember this bug again. :)

Reply:
Ah yes I remember now! :) Let's go with:

```suggestion
* udp_proxy: fixed a crash due to UDP packets being processed after listener removal.
```

?

Reply:
Ok. That is more better. :)